packages/flutter/test/widgets/framework_test.dart

```diff
@@ -501,7 +501,7 @@ void main() {
       ],
     ));
     FlutterError.onError = oldHandler;
-    expect(count, 2);
+    expect(count, 1);
```

Comment:
an additional exception was previously through because the tree building is halted have way through. Now, the glabol key reserve will not stop tree building until it finalizes.